stingray/tests/test_powerspectrum.py

```diff
@@ -53,24 +55,24 @@ def test_periodogram_types(self):
     def test_init_with_lightcurve(self):
         assert Powerspectrum(self.lc)
 
-    @raises(AssertionError)
     def test_init_without_lightcurve(self):
-        assert Powerspectrum(self.lc.counts)
+        with pytest.raises(AssertionError):
+            assert Powerspectrum(self.lc.counts)
 
-    @raises(AssertionError)
     def test_init_with_nonsense_data(self):
         nonsense_data = [None for i in range(100)]
```

Comment:
In Stingay, we use only `range` deliberately, because `xrange` will not work in Python3.